inference-engine/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -60,7 +60,7 @@ std::vector<std::string> disabledTestPatterns() {
         // TODO: Issue 43417 sporadic issue, looks like an issue in test, reproducible only on Windows platform
         R"(.*decomposition1_batch=5_hidden_size=10_input_size=30_.*tanh.relu.*_clip=0_linear_before_reset=1.*_targetDevice=CPU_.*)",
         // TODO: Sporadic Issue: 45163
-        R"(.*Behavior.*CancellationTests.*canResetAfterCancelAsyncRequest.*)",
+        // R"(.*Behavior.*CancellationTests.*canResetAfterCancelAsyncRequest.*)",
```

Comment:
we can just remove these lines.

Do we need changes in MKLDNN submodule?

Reply:
Looks like committed mkl-dnn accidentally after merging with master